libs/backend-apisix-standalone/src/transformer.ts

```diff
@@ -41,13 +41,14 @@ export const toADC = (input: typing.APISIXStandalone) => {
           description: service.desc,
           labels: service.labels,
           ...(service.upstream_id && {
-            upstream: ADCSDK.utils.recursiveOmitUndefined(
-              transformUpstream(
+            upstream: ADCSDK.utils.recursiveOmitUndefined({
+              ...transformUpstream(
                 input.upstreams!.find(
                   (item) => item.id === service.upstream_id,
                 )!,
               ),
-            ),
+              name: undefined,
```

Comment:
Fix an unexpected bug left over from the previous PR #319.

The `name` is added during sync and should be deleted during dump. Otherwise, this will cause unexpected conditions when diff resources.